project/src/main/java/com/google/sps/data/DirectoryCandidate.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** 
+ * Represents a candidate's information on the directory page.
+ */
+public class DirectoryCandidate {
+  private String id;
+  private String name;
+  private String partyAffiliation;
+  private String incumbency;
```

Comment:
nit: Would this make more sense as a bool?

Reply:
Yes, I'll make the changes. This decision was made when I mistakenly thought datastore arrays don't support boolean values